### PR TITLE
Add coverage script

### PR DESCRIPTION
**Pull request by minega25**

### CODE REVIEW

**Review by salviosage — APPROVED** [review]

**Review by nshutijonathan — COMMENTED** [review]
@minega25 The coverage badge is not working

**Comment by minega25** [comment]
> @minega25 The coverage badge is not working

if the the PR is merged travis will run tests and a coverage report will be sent to coveralls.io

**Review by habinezadalvan — APPROVED** [review]
@minega25 I think this will work because the default branch on coveralls is the develop branch and the markdown in the badge is from that branch. So hopefully when it is merged it will work. Now the after_success script is working without throwing the error. 
Screen Shot 2019-08-09 at 19 52 25

File: package.json [hunk 1]
```diff
@@ -9,13 +9,15 @@
...
-    "test": "export NODE_ENV=test && npm run undomigration && npm run migration && npm run seeds && coverage mocha ./test --recurssive --no-timeout --exit --require @babel/register",
     "dev": "npm run migration && npm run seeds && nodemon --exec babel-node ./src/app.js",
-    "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
-    "coveralls-coverage": "coveralls < lcov.info",
-    "coveralls": "npm run generate-lcov && npm run coveralls-coverage"
+    "test": "export NODE_ENV=test && npm run undomigration && npm run migration && npm run seeds && nyc mocha ./test --recurssive --no-timeout --exit --require @babel/register",
+    "coveralls": "nyc report --reporter=text-lcov | coveralls"
   },
   "author": "Andela Simulations Programme",
+  "repository": {
```
Comment (Mnickii) [inline package.json]: This is not what i posted in https://github.com/lemurheavy/coveralls-public/issues/208#issue-27138842

**Review by nshutijonathan — APPROVED** [review]

**Pull request merged**